util/convert_json_to_data.js

```diff
@@ -27,6 +27,15 @@ fs.writeFile(
     () => console.log('Updated all_data_chennai.json'),
 );
 
+const outputAp = fs.readFileSync(path.resolve(__dirname + '/output_ap.json'));
+const outputObjectAp = JSON.parse(outputAp);
```

Comment:
And could you rename these to `outputAP` and `outputObjectAP` please? Thanks, I think that will be all.